src/SFA.DAS.Commitments.Domain/Interfaces/IApprenticeshipEvents.cs

```diff
@@ -5,7 +5,6 @@
 
 namespace SFA.DAS.Commitments.Domain.Interfaces
 {
-    [Obsolete("Use IApprenticeshipEventsPublisher and IApprenticeshipEventsList.")]
```

Comment:
again why is the obsolete attribute being removed

Reply:
We've discussed this in person now, but for the record...
My previous comment addressed why I was removing [Obsolete].
Also, as previously mentioned there are 2 different services performing the same function of publishing events. The older (existing) service was marked Obsolete when the new service was written. I was going to switch the consumers over to the new service, but we need to decide which service would be best to use going forward (or combine the best of both into a new service) and there was too much work to be done for the scope of this story.